po/fr.po

```diff
@@ -16892,408 +16843,420 @@ msgstr "Changer le répertoire de travail"
 "Execute specified command for each located file, in the files directory, "
 "after asking user"
 msgstr ""
+"Exécuter la commande spécifiée pour chaque fichier trouvé, dans le dossier "
+"contenant le fichier, après accord de l’utilisateur"
 
 #: /tmp/fish/implicit/share/completions/find.fish:64
 msgid "Print null separated full file names"
-msgstr ""
+msgstr "Afficher les chemins absolus des fichiers séparés par \\0"
 
 #: /tmp/fish/implicit/share/completions/find.fish:65
-#, fuzzy
 msgid "Print formated data"
-msgstr "Affiche le texte formaté"
+msgstr "Afficher le résultat au format indiqué"
 
 #: /tmp/fish/implicit/share/completions/find.fish:66
 msgid "Do not recurse unless -depth is specified"
-msgstr ""
+msgstr "Ne pas faire de récursion sauf si -depth est spécifié"
 
 #: /tmp/fish/implicit/share/completions/find.fish:67
-#, fuzzy
 msgid "Exit at once"
-msgstr "Quitter le shell"
+msgstr "Quitter immédiatement"
 
 #: /tmp/fish/implicit/share/completions/find.fish:68
-#, fuzzy
 msgid "List file in ls -dils format"
-msgstr "Lister les bogues au format rss"
+msgstr "Lister les fichiers au format ls -dils"
 
 #: /tmp/fish/implicit/share/completions/find.fish:69
-#, fuzzy
 msgid "Negate result of action"
-msgstr "Inverser l'état de sortie de la tâche"
+msgstr "Inverser le résultat de l’action"
 
 #: /tmp/fish/implicit/share/completions/find.fish:70
 msgid "Result is only true if both previous and next action are true"
 msgstr ""
+"Le résultat n’est vrai que si les actions précédente et suivante sont toutes "
+"deux vraies"
 
 #: /tmp/fish/implicit/share/completions/find.fish:71
 msgid "Result is true if either previous or next action are true"
-msgstr ""
+msgstr "Le résultat est vrai si l’action précédente ou suivante est vraie"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:1
-#, fuzzy
 msgid "Run fish with this command"
-msgstr "Définir ou obtenir la ligne de commande"
+msgstr "Exécuter la commande spécifiée au lieu d’afficher l’invite"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:4
 msgid "Only parse input, do not execute"
 msgstr ""
+"Ne faire que la vérification syntaxique de la commande, pas son exécution"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:5
-#, fuzzy
 msgid "Run in interactive mode"
-msgstr "Exécuter en mode noninteractif"
+msgstr "Exécuter en mode interactif"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:6
-#, fuzzy
 msgid "Run in login mode"
-msgstr "Exécuter en mode fictif"
+msgstr "Exécuter fish comme un shell de connexion"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:7
-#, fuzzy
 msgid "Output profiling information to specified file"
-msgstr "Écriture des informations de profilage dans le fichier '%s' impossible"
+msgstr "Écriture des informations de profilage dans le fichier spécifié"
 
 #: /tmp/fish/implicit/share/completions/fish.fish:8
-#, fuzzy
 msgid "Run with the specified verbosity level"
-msgstr "Fichier sources.list de sortie"
+msgstr "Exécuter avec la jactance spécifiée"
```

Comment:
Inconsistent terminology for verbose / verbos(e|ity) level:
- line 3731: “be verbose” → ”être bavard” (literally “be talkative”)
- line 16199:  “verbose messages” → ”messages détaillés” (literally “detailed messages”)
- current line, line 24670: “verbos(e|ity) level” → ”jactance” (literally “verbal display of arrogance” – at least in Metropolitan French, see TLFI; it may have a different connotation in a tech context, or in _Québécois_)

Reply:
In fact, *verbosity* has no true synonym in French, at least none I know, but *jactance* is a popular term for *babble*, so it seems appropriate as a replacement, even if it is not as formal.

Reply:
I don't speak French, but verbosité?

Reply:
Didn't see this one; yet, it looks like an anglicism, despite not being one, so I'm unsure it would be more accepted.

Reply:
_Verbosité_ has been a member in good standing of the French language since at least the 15th century, says the [TLFi](http://stella.atilf.fr/Dendien/scripts/tlfiv5/advanced.exe?8;s=2222671035;). So, no, most certainly not an anglicism. As it is a far more precise translation than _jactance_, I would advocate using it.

Reply:
I did not say it was an anglicism, only that it sounded like one; I'll do with it, then.